### ISSUE TITLE
Integrators

### ISSUE THREAD

**Issue by mottosso** [pyblish/pyblish-base#82]
For the 1.1, as is being discussed in the [Usergroup](https://groups.google.com/forum/#!topic/pyblish/CgHkv2jmLhk), we're thinking about renaming Conformer to Integrator.

This issue is part of the 1.1 "milestone". Let's tag each issue relevant to the 1.1 with this milestone, that way, we can spot progress and keep track of exactly what we are up against.

**Comment by mottosso** [comment]
This has been implemented in the form of an alias, since [1.0.1](https://github.com/pyblish/pyblish/tree/1a379aca33ab36840271541a51334d9feb7cc9f0).

``` python
import pyblish.api as pyblish

class IntegrateAsset(pyblish.Integrator):
  ...
```

However bear in mind that the physical files will still need to be named `conform_`. 

Considering whether both should be allowed?

``` bash
# E.g.
conform_asset.py
integrate_asset.py
```

**Comment by tokejepsen** [comment]
I'm actually warming to conforming:)

**Comment by mottosso** [comment]
:)

**Comment by mkolar** [comment]
I quite like conform too. But quite frankly renaming it doesn't bother me in the slightest. Both work equally good as far as I can tell.

**Comment by mottosso** [comment]
Just to clarify, there won't be any renaming. This would simply add another option to how you are able to name the files on disk. Either prefixed with `conform_` or `integrate_` depending on your preference.

You could potentially use both, depending on whether one suits better than the other.

```
conform_images.py
integrate_asset.py
```

**Comment by mottosso** [comment]
Implemented in 1.1
